sdk/src/dlob/orderBookLevels.ts

```diff
@@ -88,9 +88,14 @@ export function* getL2GeneratorFromDLOBNodes(
 	slot: number
 ): Generator<L2Level> {
 	for (const dlobNode of dlobNodes) {
-		const size = dlobNode.order.baseAssetAmount.sub(
+		const size = dlobNode.baseAssetAmount.sub(
 			dlobNode.order.baseAssetAmountFilled
 		) as BN;
```

Comment:
can the base adjustment for reduce only orders cause this to flip? might be safer to max(0) here